examples/cosmos3_story_scene/app/static/styles.css

```
@@ -0,0 +1,1170 @@
+  --quiet: #717a74;
+  --line: rgba(244, 247, 242, 0.13);
+  --line-strong: rgba(244, 247, 242, 0.26);
+  --green: #b7f52e;
+  --green-deep: #7cab12;
+  --cyan: #6ed8db;
+  --blue: #72a8ff;
+  --orange: #ff9d5c;
+  --red: #ff7068;
+  --danger: #ff958d;
+  --success: #98f5b4;
+  --sans: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)